src/common/client.ts

```diff
@@ -29,26 +39,77 @@ let clientInstance: Client | null = null;
  */
 export function getClient(): Client {
   if (!clientInstance) {
-    const subdomain = process.env.GLEAN_SUBDOMAIN;
-    const token = process.env.GLEAN_API_TOKEN;
+    const glean = new Glean(getSDKOptions());
 
-    if (!subdomain) {
-      throw new Error('GLEAN_SUBDOMAIN environment variable is required');
-    }
+    clientInstance = glean.client;
+  }
+
+  return clientInstance;
+}
+
+function buildHttpClientWithGlobalHeaders(
+  headers: Record<string, string>,
+): HTTPClient {
+  const httpClient = new HTTPClient({
+    fetcher: (request) => {
+      return fetch(request);
+    },
```

Comment:
Can you just do:

```suggestion
    fetcher: fetch;
    },
```

Or better yet, nothing?

Reply:
Yeah good call we can rely on the default fetcher.